doc/ReleaseNotes

```diff
@@ -31,6 +31,16 @@ TCPDirect distributed as source code
   packages are suitable for all supported OSs.
 
 
+New onload-devel package required
+---------------------------------
+
+  Onload now includes a package containing all of the header files used by
+  Onload. Installing this package is now required to build TCPDirect applications.
```

Comment:
Should this be `TCPDirect binaries` instead (or as well as) `TCPDirect applications` ?

Reply:
Good point (as well as). But the reasoning for it is before the install steps so maybe need to mention the TCPDirect binaries on priority